lib/output.js

```diff
@@ -597,6 +598,12 @@ function OutputStream(options) {
             return true;
         }
 
+        if (output.option('webkit')
+            && output.parent(0) instanceof AST_PropAccess
+            && output.parent(1) instanceof AST_Assign) {
```

Comment:
> Thanks for putting up with me. I was just making a case for less code.

You are always welcome :wink: 

I just need to let some steam out - both me and my box do not thrive under 35℃ & 100% humidity.